src/coreComponents/physicsSolvers/wavePropagation/AcousticWaveEquationSEM.cpp

```diff
@@ -226,12 +228,19 @@ void AcousticWaveEquationSEM::postProcessInput()
   {
     m_nsamplesSeismoTrace = 0;
   }
-  localIndex const nsamples = int(maxTime/dt) + 1;
+  localIndex const nsamples = int((maxTime - minTime)/dt) + 1;
+
+
+  localIndex numNodesPerElem = getNumNodesPerElem();
 
   localIndex const numSourcesGlobal = m_sourceCoordinates.size( 0 );
+  m_sourceNodeIds.resize( numSourcesGlobal, numNodesPerElem );
+  m_sourceConstants.resize( numSourcesGlobal, numNodesPerElem );
   m_sourceIsAccessible.resize( numSourcesGlobal );
 
   localIndex const numReceiversGlobal = m_receiverCoordinates.size( 0 );
+  m_receiverNodeIds.resize( numReceiversGlobal, numNodesPerElem );
+  m_receiverConstants.resize( numReceiversGlobal, numNodesPerElem );
   m_receiverIsLocal.resize( numReceiversGlobal );
 
```

Comment:
Same as before you can remove the arrays resized inside  initializePreSubGroups, norw it should work even with pygeosx as initializePreSubGroups is called in reinit so you could remove the duplicated resize (duplicated between initializePreSubGroups and postProcessInput) 
You can also remove "getNumNodesPerElem" call, it gives 0 here (found it last time...), it is anly useful in initializePreSubGroups